public/static/gameScene.js

```diff
@@ -447,6 +471,38 @@ class GameScene extends Phaser.Scene {
 
     }
 
+    displayMissileCount(self, id, positionX, newAmount, maxAmount, regenTime) {
+        const positionY = 575;
+        const maxDisplayTimeMs = 5000;
+        let myGroup = this.add.group();
+        self.missileCounterUIs[id] = myGroup;
+        const missileSprite = self.add.sprite(positionX - 45, positionY, 'missile').setDisplaySize(20,30).setDepth(100);
+        const amountText = self.add.text(positionX - 15, positionY, '' + newAmount + '/' + maxAmount, { fontSize: '24px' }).setTint(0xffffff).setDepth(100);
+        myGroup.add(missileSprite);
+        myGroup.add(amountText);
+        self.missileCounterUIs[id] = myGroup;
```

Comment:
Isn't this line just a duplicate of the line in 478?

Reply:
oops, yeah it is. deleting 478

Reply:
gone